2014/01/16 22:16:42 [error] [exception.CException] exception 'CException' with message 'CAssetManager.basePath "C:\Apache24\htdocs\yii\training\assets" is invalid. Please make sure the directory exists and is writable by the Web server process.' in C:\Apache24\htdocs\yii\framework\web\CAssetManager.php:138
Stack trace:
#0 C:\Apache24\htdocs\yii\framework\web\CAssetManager.php(123): CAssetManager->setBasePath('C:\Apache24\htd...')
#1 C:\Apache24\htdocs\yii\framework\web\CAssetManager.php(217): CAssetManager->getBasePath()
#2 C:\Apache24\htdocs\yii\framework\web\CClientScript.php(510): CAssetManager->publish('C:\Apache24\htd...')
#3 C:\Apache24\htdocs\yii\framework\web\CClientScript.php(547): CClientScript->getCoreScriptUrl()
#4 C:\Apache24\htdocs\yii\framework\web\CClientScript.php(348): CClientScript->getPackageBaseUrl('jquery')
#5 C:\Apache24\htdocs\yii\framework\web\CClientScript.php(208): CClientScript->renderCoreScripts()
#6 C:\Apache24\htdocs\yii\framework\web\CController.php(351): CClientScript->render('<!DOCTYPE html ...')
#7 C:\Apache24\htdocs\yii\framework\web\CController.php(788): CController->processOutput('<!DOCTYPE html ...')
#8 C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php(72): CController->render('contact', Array)
#9 C:\Apache24\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionContact()
#10 C:\Apache24\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\Apache24\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\Apache24\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('contact')
#14 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contact')
#15 C:\Apache24\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#16 C:\Apache24\htdocs\yii\training\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/index.php?r=site/contact
HTTP_REFERER=http://training/index.php?r=site/page&view=about
---
2014/01/16 22:57:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "ajax/index".' in C:\Apache24\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ajax/index')
#1 C:\Apache24\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 C:\Apache24\htdocs\yii\training\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php?r=ajax/index
HTTP_REFERER=http://training/
---
2014/01/16 23:24:57 [error] [php] mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() (C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php:67)
Stack trace:
#0 C:\Apache24\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\Apache24\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#2 C:\Apache24\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\Apache24\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\Apache24\htdocs\yii\training\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/site/contact
in C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php (67)
in C:\Apache24\htdocs\yii\training\index.php (13)
2014/01/16 23:28:01 [error] [php] mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() (C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php:67)
Stack trace:
#0 C:\Apache24\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\Apache24\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#2 C:\Apache24\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\Apache24\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#6 C:\Apache24\htdocs\yii\training\index.php(13): CWebApplication->run()
REQUEST_URI=/index.php/site/contact
in C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php (67)
in C:\Apache24\htdocs\yii\training\index.php (13)
2014/02/20 01:02:46 [error] [php] Undefined index: test (C:\Apache24\htdocs\yii\training\protected\views\site\index.php:17)
Stack trace:
#0 C:\Apache24\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\Apache24\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php(16): SiteController->render()
#3 C:\Apache24\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\Apache24\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\Apache24\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\Apache24\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\Apache24\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\Apache24\htdocs\yii\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 C:\Apache24\htdocs\yii\training\index.php(13): CWebApplication->run()
REQUEST_URI=/
in C:\Apache24\htdocs\yii\training\protected\views\site\index.php (17)
in C:\Apache24\htdocs\yii\training\protected\controllers\SiteController.php (16)
in C:\Apache24\htdocs\yii\training\index.php (13)
2014/03/04 12:27:52 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.
' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(438): CModule->getComponent('db')
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(623): CApplication->getDb()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(2309): CActiveRecord->getDbConnection()
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Units))
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(402): CActiveRecord::model('Units')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(77): CActiveRecord->getMetaData()
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(37): CActiveRecord->__construct()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#13 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#14 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#15 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#16 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#18 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:33:00 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'training.units' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `Units`.
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (37)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:33:00 [error] [exception.CDbException] exception 'CDbException' with message 'The table "Units" for active record class "Units" cannot be found in the database.' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php:2310
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(387): CActiveRecordMetaData->__construct(Object(Units))
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(402): CActiveRecord::model('Units')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(77): CActiveRecord->getMetaData()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(37): CActiveRecord->__construct()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:34:18 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (43)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:34:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(43): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:42:18 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (43)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:42:18 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(43): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:44:14 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:44:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(46): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:44:31 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:44:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(46): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:45:02 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'count' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`text`) VALUES (:yp0).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:45:02 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'count' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`text`) VALUES (:yp0)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(46): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:46:16 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'count' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`text`) VALUES (:yp0).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:46:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'count' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`text`) VALUES (:yp0)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(46): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 12:48:51 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (47)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 12:48:51 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(47): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 13:15:39 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bout' for key 'type_UNIQUE'. The SQL statement executed was: INSERT INTO `Units` (`text`, `count`, `type`) VALUES (:yp0, :yp1, :yp2).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (43)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 13:15:39 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bout' for key 'type_UNIQUE'. The SQL statement executed was: INSERT INTO `Units` (`text`, `count`, `type`) VALUES (:yp0, :yp1, :yp2)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(43): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 13:15:47 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bout' for key 'type_UNIQUE'. The SQL statement executed was: INSERT INTO `Units` (`text`, `count`, `type`) VALUES (:yp0, :yp1, :yp2).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (43)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 13:15:47 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'bout' for key 'type_UNIQUE'. The SQL statement executed was: INSERT INTO `Units` (`text`, `count`, `type`) VALUES (:yp0, :yp1, :yp2)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(43): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 13:18:10 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (45)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/04 13:18:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php(45): CActiveRecord->save()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionAddUnit()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('addunit')
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/addunit')
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/index.php/site/addunit/
HTTP_REFERER=http://training/
---
2014/03/04 16:44:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 16:44:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 16:45:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 18:44:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "index.php/site/index".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('index.php/site/...')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/site/index
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:07:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:08:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:08:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:09:22 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:09:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "index.php".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php:483
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(270): CController->missingAction('index.php')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): CController->run('index.php')
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index.php/...')
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/index.php/site/index.php/site/addunit/
HTTP_REFERER=http://training/index.php/site/index
---
2014/03/04 19:18:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "http://training".' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('http://training')
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/index.php/http://training
HTTP_REFERER=http://training/
---
2014/03/05 16:11:36 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 16:12:03 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 16:42:10 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (46)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 16:53:32 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (47)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 16:54:22 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (47)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 16:59:58 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[HY000]: General error: 1364 Field 'text' doesn't have a default value. The SQL statement executed was: INSERT INTO `Units` (`id`) VALUES (NULL).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (47)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (13)
2014/03/05 23:06:33 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL int(10) unsigned NOT NULL AUTO_INCREMENT,
	"text" varchar(100) NOT NULL,
	"count" int(3) unsigned NOT NULL,
	"type" varchar(20) NOT NULL
)
2014/03/05 23:06:33 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 near "int": syntax error. The SQL statement executed was: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL int(10) unsigned NOT NULL AUTO_INCREMENT,
	"text" varchar(100) NOT NULL,
	"count" int(3) unsigned NOT NULL,
	"type" varchar(20) NOT NULL
).
2014/03/05 23:06:33 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 near "int": syntax error. The SQL statement executed was: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL int(10) unsigned NOT NULL AUTO_INCREMENT,
	"text" varchar(100) NOT NULL,
	"count" int(3) unsigned NOT NULL,
	"type" varchar(20) NOT NULL
)' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(1352): CDbCommand->execute()
#1 /opt/lampp/htdocs/yii/framework/db/CDbMigration.php(250): CDbCommand->createTable('Units', Array, NULL)
#2 /opt/lampp/htdocs/yii/training/protected/migrations/m140305_174906_CreateTableUnits.php(13): CDbMigration->createTable('Units', Array)
#3 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(415): m140305_174906_CreateTableUnits->up()
#4 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(109): MigrateCommand->migrateUp('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#8 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(184): CConsoleApplication->processRequest()
#10 /opt/lampp/htdocs/yii/framework/yiic.php(33): CApplication->run()
#11 /opt/lampp/htdocs/yii/training/protected/yiic.php(7): require_once('/opt/lampp/htdo...')
#12 /opt/lampp/htdocs/yii/training/protected/yiic(4): require_once('/opt/lampp/htdo...')
#13 {main}
---
2014/03/05 23:09:13 [error] [system.db.CDbCommand] Error in preparing SQL: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL AUTO_INCREMENT,
	"text" varchar(255) NOT NULL,
	"count" number NOT NULL,
	"type" varchar(255) NOT NULL
)
2014/03/05 23:09:13 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 near "AUTO_INCREMENT": syntax error. The SQL statement executed was: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL AUTO_INCREMENT,
	"text" varchar(255) NOT NULL,
	"count" number NOT NULL,
	"type" varchar(255) NOT NULL
).
2014/03/05 23:09:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 near "AUTO_INCREMENT": syntax error. The SQL statement executed was: CREATE TABLE 'Units' (
	"id" integer PRIMARY KEY AUTOINCREMENT NOT NULL AUTO_INCREMENT,
	"text" varchar(255) NOT NULL,
	"count" number NOT NULL,
	"type" varchar(255) NOT NULL
)' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(1352): CDbCommand->execute()
#1 /opt/lampp/htdocs/yii/framework/db/CDbMigration.php(250): CDbCommand->createTable('Units', Array, NULL)
#2 /opt/lampp/htdocs/yii/training/protected/migrations/m140305_174906_CreateTableUnits.php(13): CDbMigration->createTable('Units', Array)
#3 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(415): m140305_174906_CreateTableUnits->up()
#4 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(109): MigrateCommand->migrateUp('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#8 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(184): CConsoleApplication->processRequest()
#10 /opt/lampp/htdocs/yii/framework/yiic.php(33): CApplication->run()
#11 /opt/lampp/htdocs/yii/training/protected/yiic.php(7): require_once('/opt/lampp/htdo...')
#12 /opt/lampp/htdocs/yii/training/protected/yiic(4): require_once('/opt/lampp/htdo...')
#13 {main}
---
2014/03/05 23:21:13 [error] [system.db.CDbCommand] Error in preparing SQL: DROP TABLE 'Users'
2014/03/05 23:21:13 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such table: Users. The SQL statement executed was: DROP TABLE 'Users'.
2014/03/05 23:21:13 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such table: Users. The SQL statement executed was: DROP TABLE 'Users'' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(1375): CDbCommand->execute()
#1 /opt/lampp/htdocs/yii/framework/db/CDbMigration.php(275): CDbCommand->dropTable('Users')
#2 /opt/lampp/htdocs/yii/training/protected/migrations/m140305_174906_CreateTableUnits.php(18): CDbMigration->dropTable('Users')
#3 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(440): m140305_174906_CreateTableUnits->down()
#4 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(181): MigrateCommand->migrateDown('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionRedo(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#8 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(184): CConsoleApplication->processRequest()
#10 /opt/lampp/htdocs/yii/framework/yiic.php(33): CApplication->run()
#11 /opt/lampp/htdocs/yii/training/protected/yiic.php(7): require_once('/opt/lampp/htdo...')
#12 /opt/lampp/htdocs/yii/training/protected/yiic(4): require_once('/opt/lampp/htdo...')
#13 {main}
---
2014/03/05 23:21:50 [error] [system.db.CDbCommand] Error in preparing SQL: DROP TABLE 'Users'
2014/03/05 23:21:50 [error] [system.db.CDbCommand] CDbCommand::execute() failed: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such table: Users. The SQL statement executed was: DROP TABLE 'Users'.
2014/03/05 23:21:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: CDbCommand failed to prepare the SQL statement: SQLSTATE[HY000]: General error: 1 no such table: Users. The SQL statement executed was: DROP TABLE 'Users'' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(1375): CDbCommand->execute()
#1 /opt/lampp/htdocs/yii/framework/db/CDbMigration.php(275): CDbCommand->dropTable('Users')
#2 /opt/lampp/htdocs/yii/training/protected/migrations/m140305_174906_CreateTableUnits.php(18): CDbMigration->dropTable('Users')
#3 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(440): m140305_174906_CreateTableUnits->down()
#4 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(181): MigrateCommand->migrateDown('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionRedo(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#8 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(184): CConsoleApplication->processRequest()
#10 /opt/lampp/htdocs/yii/framework/yiic.php(33): CApplication->run()
#11 /opt/lampp/htdocs/yii/training/protected/yiic.php(7): require_once('/opt/lampp/htdo...')
#12 /opt/lampp/htdocs/yii/training/protected/yiic(4): require_once('/opt/lampp/htdo...')
#13 {main}
---
2014/03/05 23:30:01 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'training.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
2014/03/05 23:30:04 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'number NOT NULL,
	`type` varchar(255) NOT NULL
)' at line 4. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` number NOT NULL,
	`type` varchar(255) NOT NULL
).
2014/03/05 23:30:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'number NOT NULL,
	`type` varchar(255) NOT NULL
)' at line 4. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` number NOT NULL,
	`type` varchar(255) NOT NULL
)' in /opt/lampp/htdocs/yii/framework/db/CDbCommand.php:358
Stack trace:
#0 /opt/lampp/htdocs/yii/framework/db/CDbCommand.php(1352): CDbCommand->execute()
#1 /opt/lampp/htdocs/yii/framework/db/CDbMigration.php(250): CDbCommand->createTable('Units', Array, NULL)
#2 /opt/lampp/htdocs/yii/training/protected/migrations/m140305_174906_CreateTableUnits.php(13): CDbMigration->createTable('Units', Array)
#3 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(415): m140305_174906_CreateTableUnits->up()
#4 /opt/lampp/htdocs/yii/framework/cli/commands/MigrateCommand.php(109): MigrateCommand->migrateUp('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 /opt/lampp/htdocs/yii/framework/console/CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 /opt/lampp/htdocs/yii/framework/console/CConsoleCommandRunner.php(71): CConsoleCommand->run(Array)
#8 /opt/lampp/htdocs/yii/framework/console/CConsoleApplication.php(92): CConsoleCommandRunner->run(Array)
#9 /opt/lampp/htdocs/yii/framework/base/CApplication.php(184): CConsoleApplication->processRequest()
#10 /opt/lampp/htdocs/yii/framework/yiic.php(33): CApplication->run()
#11 /opt/lampp/htdocs/yii/training/protected/yiic.php(7): require_once('/opt/lampp/htdo...')
#12 /opt/lampp/htdocs/yii/training/protected/yiic(4): require_once('/opt/lampp/htdo...')
#13 {main}
---
2014/03/06 13:41:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.
' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(442): CModule->getComponent('db')
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(451): MigrateCommand->getDbConnection()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(482): MigrateCommand->getMigrationHistory(-1)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(84): MigrateCommand->getNewMigrations()
#7 [internal function]: MigrateCommand->actionUp(Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#12 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\yiic.php(33): CApplication->run()
#13 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\yiic.php(7): require_once('C:\Users\mihail...')
#14 {main}
---
2014/03/06 13:45:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.
' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(442): CModule->getComponent('db')
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(451): MigrateCommand->getDbConnection()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(482): MigrateCommand->getMigrationHistory(-1)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(84): MigrateCommand->getNewMigrations()
#7 [internal function]: MigrateCommand->actionUp(Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#12 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\yiic.php(33): CApplication->run()
#13 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\yiic.php(7): require_once('C:\Users\mihail...')
#14 {main}
---
2014/03/06 13:46:54 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение.
' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(442): CModule->getComponent('db')
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(451): MigrateCommand->getDbConnection()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(482): MigrateCommand->getMigrationHistory(-1)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(84): MigrateCommand->getNewMigrations()
#7 [internal function]: MigrateCommand->actionUp(Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#12 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\yiic.php(33): CApplication->run()
#13 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\yiic.php(7): require_once('C:\Users\mihail...')
#14 {main}
---
2014/03/06 15:50:49 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'training.tbl_migration' doesn't exist. The SQL statement executed was: SHOW FULL COLUMNS FROM `tbl_migration`.
2014/03/06 15:51:05 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'units' already exists. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` int(11) NOT NULL,
	`type` varchar(255) NOT NULL
).
2014/03/06 15:51:05 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'units' already exists. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` int(11) NOT NULL,
	`type` varchar(255) NOT NULL
)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php(1321): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbMigration.php(228): CDbCommand->createTable('Units', Array, NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\migrations\m140305_174906_CreateTableUnits.php(13): CDbMigration->createTable('Units', Array)
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(385): m140305_174906_CreateTableUnits->up()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(109): MigrateCommand->migrateUp('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\yiic.php(33): CApplication->run()
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\yiic.php(7): require_once('C:\Users\mihail...')
#12 {main}
---
2014/03/06 15:51:45 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'units' already exists. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` int(11) NOT NULL,
	`type` varchar(255) NOT NULL
).
2014/03/06 15:51:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'units' already exists. The SQL statement executed was: CREATE TABLE `Units` (
	`id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
	`text` varchar(255) NOT NULL,
	`count` int(11) NOT NULL,
	`type` varchar(255) NOT NULL
)' in C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbCommand.php(1321): CDbCommand->execute()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\db\CDbMigration.php(228): CDbCommand->createTable('Units', Array, NULL)
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\migrations\m140305_174906_CreateTableUnits.php(13): CDbMigration->createTable('Units', Array)
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(385): m140305_174906_CreateTableUnits->up()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\cli\commands\MigrateCommand.php(109): MigrateCommand->migrateUp('m140305_174906_...')
#5 [internal function]: MigrateCommand->actionUp(Array)
#6 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommand.php(172): ReflectionMethod->invokeArgs(Object(MigrateCommand), Array)
#7 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleCommandRunner.php(67): CConsoleCommand->run(Array)
#8 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\console\CConsoleApplication.php(91): CConsoleCommandRunner->run(Array)
#9 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CConsoleApplication->processRequest()
#10 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\yiic.php(33): CApplication->run()
#11 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\yiic.php(7): require_once('C:\Users\mihail...')
#12 {main}
---
2014/03/06 18:47:20 [error] [system.db.CDbCommand] CDbCommand::execute() failed: SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '06/03/2014' for column 'date_create' at row 1. The SQL statement executed was: INSERT INTO `Units` (`text`, `count`, `type`, `date_create`) VALUES (:yp0, :yp1, :yp2, :yp3).
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (47)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (12)
2014/03/07 09:11:07 [error] [php] Object of class DateTime could not be converted to string (C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php:49)
Stack trace:
#0 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\Users\mihail\Documents\NetBeansProjects\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php(12): CWebApplication->run()
REQUEST_URI=/index.php/site/addUnit
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\protected\controllers\SiteController.php (49)
in C:\Users\mihail\Documents\NetBeansProjects\yii\Training\index.php (12)
